### PR TITLE
Fix bulk queue_size and size in thread pool document

### PR DESCRIPTION
**Pull request by wyukawa**
I check source code https://github.com/elastic/elasticsearch/blob/v5.3.0/core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java#L172

I conclude current document contains bugs.

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/modules/threadpool.asciidoc [hunk 1]
```diff
@@ -75,7 +75,7 @@ requests with a queue (optionally bounded) for pending requests that
 have no threads to service them.
 
 The `size` parameter controls the number of threads, and defaults to the
-number of cores times 5.
+number of cores.
```
Comment (jasontedor) [inline docs/reference/modules/threadpool.asciidoc]: Let's revert this change, I do not think it's correct anyway any longer, there is no default for a fixed thread pool, it must be specified on construction of the thread pool and each thread pool is free to set its own default. Fixing this should be done in a separate change.

**Review by jasontedor — CHANGES_REQUESTED** [review]
Thanks so much for noticing this and submitting a PR. The queue size fix for the bulk thread pool is correct, and this will need to be backported to 5.3, 5.x, and master (we can do this when we integrate, no need for a separate PR). However, the change to the default number of thread for a fixed thread pool should be reverted. After you make that correction, we can integrate.

File: docs/reference/modules/threadpool.asciidoc (hunk 1 above)
Comment (wyukawa) [inline docs/reference/modules/threadpool.asciidoc]: Thank you for comments!

I fixed in bbdffc60a1dca67549b43a4e5052ee3d642aa1c5

**Review by jasontedor — APPROVED** [review]
LGTM. Thanks again for noticing this and fixing it!

**Comment by wyukawa** [comment]
Thanks!

**Pull request merged**